scrapyjs/middleware.py

```diff
@@ -32,6 +33,14 @@ def __init__(self, crawler, splash_base_url, slot_policy):
         self.splash_base_url = splash_base_url
         self.slot_policy = slot_policy
 
+    def get_splash_options(self, request, spider):
+        if request.meta.get("dont_proxy"):
```

Comment:
> I think explicit 'dont_splash' would be better here.

I'm not sure. But it seems like this is just a question of terminology, "dont_proxy" to my ears sounds like general signal - 'dont use any kind of proxy mechanism for this request' so I thought we could reuse it here. On the other hand "splash" here is not used as proxy so it may be confusing, also `dont_splash` may be more specific and explicit. So maybe `dont_splash` would better.